src/homepageExperience/components/steps/arduino/ExecuteQuery.tsx

```diff
@@ -30,5 +84,19 @@ export const ExecuteQuery = (props: OwnProps) => {
     return () => document.removeEventListener('keydown', fireKeyboardCopyEvent)
   }, [])
 
-  return <h1>Execute a Flux Query on {bucket}</h1>
+  return (
+    <>
+      <h1>Execute a Flux Query on {bucket}</h1>
```

Comment:
Similar comment as this:

https://github.com/influxdata/ui/pull/5401#discussion_r946051369